lib/metadata/BucketInfo.js

```diff
@@ -116,10 +138,12 @@ export default class BucketInfo {
      */
     static deSerialize(stringBucket) {
         const obj = JSON.parse(stringBucket);
+        const websiteConfig =
```

Comment:
Please change to:

`` const websiteConfig = obj.websiteConfiguration ?
 new WebsiteConfiguration(obj.websiteConfiguration) : null;``

otherwise, we get different behavior from the file backend versus in memory since deserialized is called on the file backend and this creates a website config object whereas in memory has null.